
{'date': 'Mon Nov 11 2019 10:38:37.759 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  self.ajuda = Elemento (FOCO, x=80, y=500, cena=self,fiocruz)
                                                             ^
SyntaxError: non-keyword arg after keyword arg
'''},
{'date': 'Mon Nov 11 2019 10:41:28.304 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  """O personagem dá uma explicação  de como encontrar o lab """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 11 2019 10:41:39.708 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  ""O personagem dá uma explicação  de como encontrar o lab ""
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 11 2019 10:42:11.746 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 18
  ""O personagem dá uma explicação  de como encontrar o lab""
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 11 2019 10:42:29.556 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 18
  """O personagem dá uma explicação  de como encontrar o lab"""
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 11 2019 10:44:52.347 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 18
  """O personagem dá uma explicação  de como encontrar o lab"""
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 11 2019 10:47:47.75 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    fc = FioCruz()
  module <module> line 15
    self.ajuda = Elemento(FOCO, x=30, y=350, cena=self.fiocruz, vai=self._ajuda)
AttributeError: 'FioCruz' object has no attribute '_ajuda'
'''},
{'date': 'Mon Nov 11 2019 10:55:42.14 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  self.ajuda = Elemento(FOCO, x=30, y=350, cena=self.fiocruz
                                                                           ^
SyntaxError: invalid syntax
'''},